templates/includes/records/hierarchy-global.html

```diff
@@ -101,10 +98,7 @@ <h2 class="hierarchy-full-panel__heading" id="detailed-hierarchy-title">Detailed
                 {%endfor%}
             {% else %}
                 <div class="hierarchy-full-panel__bar-container-outer">
-                    <div class="hierarchy-full-panel__bar-container-inner hierarchy-full-panel__bar-container-inner--active hierarchy-full-panel__bar-container-inner--level-2">
-                        <p><span class="hierarchy-full-panel__bar-number">N</span>
-                    <span class="hierarchy-full-panel__bar-label">records</span></p>
```

Comment:
same comments as TODO above.

Reply:
I think in this case having the code commented out would be useful for future development.